src/libs/auth.ts

```diff
@@ -302,14 +334,16 @@ authStore.subscribe(
   withErrorReporting('Error checking registration', async (state, oldState) => {
     const getRegistrationStatus = async () => {
       try {
-        const { enabled } = await Ajax().User.getStatus();
+        const { enabled } = await Ajax({}).User.getStatus();
         if (enabled) {
           // When Terra is first loaded, termsOfService.permitsSystemUsage will be undefined while the user's ToS status is fetched from Sam
-          return state.termsOfService.permitsSystemUsage ? userStatus.registeredWithTos : userStatus.registeredWithoutTos;
+          return state.termsOfService.permitsSystemUsage
+            ? userStatus.registeredWithTos
+            : userStatus.registeredWithoutTos;
         }
         return userStatus.disabled;
       } catch (error) {
-        if (error.status === 404) {
+        if ((error as any).status === 404) {
```

Comment:
```suggestion
        if ((error as Response).status === 404) {
```

Slight preference for a more specific cast here. [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) is a type we'd expect Ajax methods might throw.